lib/src/test/java/com/auth0/jwt/JWTTest.java

```diff
@@ -272,7 +273,7 @@ public void shouldGetStringAudience() throws Exception {
     public void shouldGetExpirationTime() throws Exception {
         Date expectedDate = new Date(1477592 * 1000);
         Clock clock = mock(Clock.class);
-        when(clock.getToday()).thenReturn(expectedDate);
+        when(clock.getNow()).thenReturn(expectedDate.toInstant());
```

Comment:
I'm having second thoughts with these 3 tests. You've kept them since this is (soon to be) deprecated behavior and you want to check the Date stuff still works. If that's the case then why changing `getToday` to `getNow`? As I see it, this should keep using Date stuff. 

If not, then refactor `expectedDate` so that it is an instant instead of a date. Applies to all 3 (legacy tests) below.

Reply:
Mocking `getNow` here since that is what the implementation in `JWTVerifier` uses. I will refactor `expectedDate` to be an instant and add a comment about why we are mocking `getNow` instead of `getToday` here.